VMEncryption/main/patch/UbuntuPatching.py

```diff
@@ -64,7 +64,7 @@ def install_extras(self):
         cmd = " ".join(['apt-get', 'update'])
         self.command_executor.Execute(cmd)
 
-        packages = ['at', 'cryptsetup-bin', 'lsscsi', 'python-six', 'python-parted', 'procps', 'psmisc', 'gcc', 'libssl-dev', 'libffi-dev', 'python-dev', 'python-pip']
+        packages = ['at', 'cryptsetup-bin', 'lsscsi', 'python-six', 'python-parted', 'procps', 'psmisc', 'libssl-dev', 'libffi-dev', 'python-dev']
```

Comment:
change formatting to have package names on new lines like other distro code.

Reply:
reformatted